src/layouts/KnownIncompatibilities.astro

```diff
@@ -0,0 +1,58 @@
+---
+import Page from "./Page.astro";
+
+import rawData from "../../public/api/v1/incompatible-mods.json";
+import IncompatibleModCard from "../components/atoms/IncompatibleModCard.astro";
+
+const TYPE_TO_HUMAN = {
+	"GAME": "Breaks the game",
+	"OTHERS": "Breaks other mods",
+	"WORKAROUND": "Requires a workaround",
+	"SELF": "Non-fatal errors or broken features"
+}
+
+const props = Astro.props;
+const data = rawData.sort((a, b) => a.name.localeCompare(b.name))
+---
+<Page content={props.content} frontmatter={props.frontmatter}>
+	<p>
+		This page collects all Fabric mods that have reported incompatibilities with Quilt.
+		You can hover the symbol on the left to see the current status of the issue, and click the link icon to go to the tracking issue.
+		If you believe this list is missing anything, please feel free to tell us <a href="https://discord.quiltmc.org">on Discord</a>.
+		You can also directly <a href="https://github.com/QuiltMC/quiltmc.org/blob/main/public/api/v1/incompatible-mods.json">update the data on GitHub</a>.
+	</p>
+
+	<h2>Requires a workaround</h2>
+	<p>This is the simplest form of problems, the issue can simply be fixed by following the instructions provided.</p>
+	{
+		data.find(mod => mod.type === "WORKAROUND") ? data.filter(mod => mod.type === "WORKAROUND").map((mod) => (
+			<IncompatibleModCard mod={mod} />
+		)) : (<div class="card"><div class="card-content">Currently nothing to report!</div></div>)
+	}
+
+	<h2>Non-fatal errors or broken features</h2>
+	<p>These mods are not compatible with Quilt, but they do not break the game or other mods. They may have broken features or other errors that do not crash the game.</p>
+	{
+		data.find(mod => mod.type === "SELF") ? data.filter(mod => mod.type === "SELF").map((mod) => (
+			<IncompatibleModCard mod={mod} />
+		)) : (<div class="card"><div class="card-content">Currently nothing to report!</div></div>)
+	}
+
+	<h2>Breaks other mods</h2>
+	<p>These mods are not compatible with Quilt, and they break other mods.</p>
+	{
+		data.find(mod => mod.type === "OTHERS") ? data.filter(mod => mod.type === "OTHERS").map((mod) => (
+			<IncompatibleModCard mod={mod} />
+		)) : (<div class="card"><div class="card-content">Currently nothing to report!</div></div>)
+	}
+
+	<h2>Breaks the game</h2>
+	<p>These mods are not compatible with Quilt, and they break the game.</p>
+	{
+		data.find(mod => mod.type === "GAME") ? data.filter(mod => mod.type === "GAME").map((mod) => (
+			<IncompatibleModCard mod={mod} />
+		)) : (<div class="card"><div class="card-content">Currently nothing to report!</div></div>)
+	}
+
+	<p>This data is also available in machine-readable format at <a href="/api/v1/incompatible-mods.json" target="_blank"><code>/api/v1/incompatible-mods.json</code></a>.</p>
```

Comment:
I know the structure is fairly trivial to understand, but could we maybe describe the schema somewhere, considering it's presented as an API ?

Reply:
We could and probably should provide openapi specs and a generated html page, but that's its own project and doesn't belong here